src/main/java/uk/gov/ons/census/casesvc/service/UacService.java

```diff
@@ -85,6 +85,10 @@ public UacQidLink createUacQidLinkedToCCSCase(Case caze) {
   }
 
   public PayloadDTO saveAndEmitUacUpdatedEvent(UacQidLink uacQidLink) {
```

Comment:
With the new changes this won't pass in unreceipted. just making a comment so i don't forget about it